src/components/home/ProjectSection/Project.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
+import Card from 'react-bootstrap/Card';
+import "./Project.css";
+
+type Props = {
+    name: string;
+    desc: string;
+    imageUrl: string;
+};
+
+class Project extends Component<Props>{
+    constructor(props: Props) {
+        super(props)
+    }
+    
+    render(){
+        return(
+            <Card className="project" id={this.props.name === "Hate Crime Tracker" ? "project-top" : "project-bottom"}>
```

Comment:
May I ask why do we need different project ids? If they are needed, can the caller component define and pass it in here?

Reply:
I'm using different ids mainly for css definitions, and yes using props would be better. Will change it

Reply:
FYI the reason for using different css rules was that the project on the top has slightly different image and body ratios than the two on the bottom, as well as the overall height. I have modified the code to pass the ids in using props, but the code in the caller component (ProjectSection) looks a bit tedious now. Please let me know what you think.